src/main/scss/base/_style.scss

```diff
@@ -389,6 +391,24 @@ pre.console {
   p:last-of-type {
```

Comment:
I tried removing this as its the cause of the heading spacing issue but I couldn't find a way to make it work whilst keeping no spacing at the bottom of the help file (made worse in help from plugins that have `from plugin ...` as the space was a bit excessive without this.